lesson2/task6_bonus/main.cpp

```diff
@@ -6,11 +6,16 @@
 using namespace std;
 
 int main() {
+    constexpr double Pi() { 22.0 / 7 };
```

Comment:
you cannot declare function inside another function. Here works optimization but if it does not work your code will not compile